packages/oid4vci-nestjs/src/oid4vci.service.ts

```
@@ -0,0 +1,21 @@
+import { Injectable } from '@nestjs/common';
+import {
+  CredentialRequestDto,
+  DeferredCredentialRequestDto,
+  NotificationRequestDto,
+  TokenDto,
+} from '@vdcs/oid4vci';
+
+@Injectable()
+export abstract class Oid4VciService {
+  abstract handleCredentialRequest(dto: CredentialRequestDto): Promise<any>;
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)